test/examples.js

```diff
@@ -0,0 +1,487 @@
+'use strict';
```

Comment:
I think we should split this file into appropriate tests as it's not `examples` anymore. These are just tests.

Reply:
Then we get tests similar to those already in the ``/test``. Maybe then just delete this file? Or is it better to split?

Reply:
I'd prefer to leave the tests but put them into appropriate (or newly created) files.

Though if this file only duplicates what is tested in other files then maybe we shouldn't have made a 'test' out of it and left it as is - with logs and stuff?

Reply:
As for me, the file still shows the metasynk well, and it can still be called examples. However, I can decompose those tests that were not yet in existing files, and remove the rest.